common/lib/capa/capa/templates/textline.html

```diff
@@ -7,7 +7,6 @@
     <div class="text-input-dynamath_data ${doinline}" data-preprocessor="${preprocessor['class_name']}"/>
     <div class="script_placeholder" data-src="${preprocessor['script_src']}"/>
 % endif
-
```

Comment:
Very minor nit: you could remove the edits to this file from your PR so isn't linked (since it had no actual code changes).